java-dynamic-sqs-listener-api/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>java-dynamic-sqs-listener-parent</artifactId>
+        <groupId>com.jashmore</groupId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>java-dynamic-sqs-listener-api</artifactId>
+
+    <name>Java Dynamic SQS Listener - API</name>
+    <description>This contains the api functionality for the SQS Listener. This can be used for consumer to implement themselves.</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sqs</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
```

Comment:
👍 correct Lombok scope.

Reply:
Yep!  Only just realised through this that provided is the correct one